pipeline-steps/pubcloud.groovy

```diff
@@ -75,12 +75,17 @@ def cleanup(){
         )
         withEnv(["RAX_CREDS_FILE=${pyrax_cfg}"]){
           common.venvPlaybook(
-            playbooks: ['cleanup_pubcloud.yml'],
+            playbooks: ['aio-maas-cleanup.yml',
+                        'cleanup_pubcloud.yml'],
```

Comment:
Bit of a nit ... but the aio-maas-entities.yml playbook is run inside one of the maas stages. Does it matter that aio-maas-cleanup.yml is always run even if a maas stage isn't selected?